doc/howto/storage_pools.md

```diff
@@ -271,13 +315,37 @@ Storage pool my-remote-pool3 pending on member vm03
 Storage pool my-remote-pool3 created
 ```
 
+````
+````{group-tab} UI
+
+To create a storage pool in a cluster, in a clustered environment select {guilabel}`Pools` from the {guilabel}`Storage ` section of the Main navigation.
```

Comment:
```suggestion
To create a storage pool in a clustered environment select {guilabel}`Pools` from the {guilabel}`Storage ` section of the Main navigation.
```
A bit simpler?

Maybe mention this is the same as non-clustered, but you get some extra options?